### ISSUE TITLE
Alphabetically order all methods in sidebar

### ISSUE THREAD

**Issue by smk762** [KomodoPlatform/developer-docs#380]
Please alphabetically reorder all methods

_Originally posted by @artemii235 in https://github.com/KomodoPlatform/developer-docs/pull/363#discussion_r918751531_

**Comment by smk762** [comment]
@artemii235 should this reordering ignore related methods?
e.g. with strict ordering:
```
        [
          "/basic-docs/atomicdex-api-20-dev/remove_node_from_version_stat.md",
          "remove_node_from_version_stat",
        ],
        [
          "/basic-docs/atomicdex-api-20-dev/start_simple_market_maker_bot.md",
          "start_simple_market_maker_bot",
        ],
        [
          "/basic-docs/atomicdex-api-20-dev/start_version_stat_collection.md",
          "start_version_stat_collection",
        ],
        [
          "/basic-docs/atomicdex-api-20-dev/stop_simple_market_maker_bot.md",
          "stop_simple_market_maker_bot",
        ],
        [
          "/basic-docs/atomicdex-api-20-dev/stop_version_stat_collection.md",
          "stop_version_stat_collection",
        ],
```

If we ant to keep the sidebar in alpha order, I can combine related items into the same doc like it is for message signing
![image](https://user-images.githubusercontent.com/35845239/178464186-af4c6d4c-8a5a-425b-b6bc-3238d1e1afff.png)

As seen in the image above, the sidebar text is a heading, not the method string. If there is a preference, I can convert to (or retain) the method string for subheadings under the group heading.

e.g.
- Message Signing
    - sign_message
    - verify_message
- Stats Collection
    - remove_node_from_version_stat
    - start_version_stat_collection
    - stop_version_stat_collection
    - update_version_stat_collection

**Comment by gcharang** [comment]
@smk762 can this issue be closed?

**Comment by smk762** [comment]
yep, thanks. Seems there is a slight visual variation for the `message signing` section, I'll check and deal with in future PR
![image](https://user-images.githubusercontent.com/35845239/183643980-3fb424b8-045b-40b4-9f18-c9f88d0105a9.png)
